src/main/java/app/bpartners/geojobs/file/FileWriter.java

```diff
@@ -53,4 +58,29 @@ public File write(byte[] bytes, @Nullable File directory, String filename) {
       throw new ApiException(SERVER_EXCEPTION, e);
     }
   }
+
+  public Path createTempFileSecurely(String prefix, String suffix) throws IOException {
```

Comment:
refer to this implementation to create the tempFile: https://github.com/b-partners/bpartners-annotator-api/blob/preprod/src/main/java/api/bpartners/annotator/service/event/ExportTaskCreatedService.java